### PR TITLE
Adds dwarfism injector to the necropolis loot pool

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

#14719

We're rich

# Wiki Documentation

Title

# Changelog

:cl:  
tweak: Added dwarfism injector to the necropolis loot pool
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
a hidden miner <s>nerf</s> buff

**Comment by Chubbygummibear** [comment]
as a bit of spice it could maybe be like a dwarf loadout. hardhat with light, diamond tipped drill, dwarfism injector, and a mug of ale, or something to that effect?

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
either remove a shitty loot item or add more stuff for funny

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -69,6 +69,9 @@ GLOBAL_LIST_EMPTY(bloodmen_list)
 			new /obj/item/clothing/neck/necklace/memento_mori(src)
 		if(23)
 			new /obj/item/rune_scimmy(src)
+		if(24)
+			new /obj/item/dnainjector/dwarf(src)
+
```
Comment (Marmio64) [inline code/modules/mining/lavaland/necropolis_chests.dm:74]:
```suggestion
/obj/item/grenade/plastic/miningcharge/mega
/obj/item/grenade/plastic/miningcharge/mega

```
dwarves like big booms
also to stop the miner mains from crying (they're very ungrateful)

Comment (VaelophisNyx) [reply code/modules/mining/lavaland/necropolis_chests.dm:74]: Can Miners use this to get mega mining charges faster by deconstructing it

Comment (Marmio64) [reply code/modules/mining/lavaland/necropolis_chests.dm:74]: if they could do that it's still around a 4% chance to even get this from a chest, let alone that you first need to get a chest, and also that you need to disrupt your mining grindset to go topside to get the bombs deconstructed and researched

**Comment by ynot01** [comment]
> either remove a shitty loot item or add more stuff for funny

~~this was not meant to be a mining buff~~

im diversifying the loot pool

**Comment by Smudged** [comment]
Make it so only this injector gives you a permanent beard as per - 14719

Since I'm against ALL dwarfism giving you a beard already.

**Comment by ynot01** [comment]
> Make it so only this injector gives you a permanent beard as per - 14719
> 
> Since I'm against ALL dwarfism giving you a beard already.

if that gets closed maybe

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**